### DISCUSSION TITLE
Serenity for asp.net core 3.1

### DISCUSSION THREAD

**Discussion by armanarnautovic** [serenity-is/Serenity#5700] [Help]
Hello everyone,
I would like to ask kindly if someone can give me an help to solve this problem: I would like to use serenity to realize a management project, however I would not like to do it with .NET 5.0 but with .NET Core 3.1, but installing it from the marketplace for visual studio 2019 remains bound to 5.0.
Unfortunately I can't do otherwise, also I would prefer to do it with 5.0 but I have this need.

Thanks in advance

**Comment by VictorTomaili** [comment]
You can clone [serene repository](https://github.com/serenity-is/Serene) and use [this point](https://github.com/serenity-is/Serene/tree/e922bbfb345dad763f0c04d88bf01118b5916596) for a new project. This commit is latests serene version of .net 3.1 version.

**Reply by armanarnautovic** [reply]
Perfect. Forgive me for asking, but how do I then install it?

**Reply by VictorTomaili** [reply]
You can build [template](https://github.com/serenity-is/Serene/tree/e922bbfb345dad763f0c04d88bf01118b5916596/Template) for install or you can just copy Serene.Core folder and use&modify as your project. We don't recommend use older version. I didn't see any reason for don't use .net 5 version.

**Reply by armanarnautovic** [reply]
You're right. I too would like to avoid using old versions, but in my projects I have backward compatibility problems that I hope to solve later. At the moment I will adopt this solution.

Thanks a lot.
